### PR TITLE
add data for Invokers (invoketarget, invokeaction, InvokeEvent)

### PR DESCRIPTION
**Pull request by keithamus**
#### Summary

This PR drafts up the documentation for "Invokers" ([OpenUI explainer](https://open-ui.org/components/invokers.explainer/)). This is a new experimental API for enabling buttons to control interactive elements.

#### Test results and supporting details

- [Chrome Platform Status feature](https://chromestatus.com/feature/5142517058371584)
- [WebKit PR (merged)](https://github.com/WebKit/WebKit/pull/18794)
- [Chromium PR (merged)](https://chromium-review.googlesource.com/c/chromium/src/+/4926471?usp=search)
- [Mozilla PR (merged)](https://phabricator.services.mozilla.com/D189851)

#### Related issues

https://github.com/mdn/content/pull/30829

### CODE REVIEW

File: api/HTMLElement.json [hunk 1]
```diff
@@ -1567,6 +1567,66 @@
           }
         }
       },
+      "invoke_event": {
+        "_compat": {
```
Comment (queengooborg) [inline api/HTMLElement.json:1571]:
```suggestion
        "__compat": {
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Preliminary syntax error fix, will do a full review later!

**Comment by queengooborg** [comment]
It looks like you've got a number of JSON syntax errors in this PR.  Would you mind fixing those up, and then running `npm run fix` to fix auto-fixable schema/data issues?  (Note: you can run `npm test` to find any errors in the data!)

**Comment by keithamus** [comment]
Unfortunately `npm t` consumed all my machines ram before it can finish, and then it just hangs.

**Comment by queengooborg** [comment]
> Unfortunately `npm t` consumes all my machines ram before it can finish, and then it just hangs.

Sorry to hear that!  There's a lot that the linter script does in the project.

If you'd like, I'd be happy to push a commit to fix any remaining issues!

**Comment by keithamus** [comment]
@queengooborg looks like I managed to get all issues resolved :pray:

**Review by queengooborg — APPROVED** [review]

**Pull request merged**